### PR TITLE
Component from adam

### PR DESCRIPTION
**Pull request by Mahir33**

### CODE REVIEW

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+function Footer() {
```
Comment (luke-dowling) [inline src/components/Footer.jsx]:
```suggestion
const Footer = () => {
```

File: src/App.jsx [hunk 2]
```diff
@@ -1,9 +1,11 @@
 import './App.css';
+import './components/Footer'
```
Comment (luke-dowling) [inline src/App.jsx]:
```suggestion
import {Footer} from './components/Footer'
```

**Review by luke-dowling — COMMENTED** [review]
would these changes be okay

File: src/components/Footer.jsx (hunk 1 above)
Comment (Mahir33) [reply src/components/Footer.jsx]: Sure

File: src/components/Footer.jsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+
+const Footer = () => {
```
Comment (luke-dowling) [inline src/components/Footer.jsx]:
```suggestion
export const Footer = () => {
```

File: src/components/Footer.jsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+  return (
+    <>
+      <h1>footer example</h1>
+    </>
+  );
+}
+
+export default Footer;
```
Comment (luke-dowling) [inline src/components/Footer.jsx]: delete this line
```suggestion

```

**Review by luke-dowling — APPROVED** [review]

**Pull request merged**